### PR TITLE
:tada: Restrict file loading to .ipyg and .ipynb

### LINKED ISSUE #156: Open... should restrict what files can be opened
Currently Open... allows the user to open any file:
image

Should be restricted to ipyg or ipynb

### LINKED ISSUE #211: Opening file with wrong extension should prompt an error
Currently, opening a file with the wrong extension either crashes the application or raises a ValueError in the console. The error should be caught and displayed within Pyflow and without closing Pyflow

### PR DESCRIPTION
**Pull request by FabienRoger**
Fixes #156

### CODE REVIEW

File: pyflow/graphics/window.py [hunk 1]
```diff
@@ -292,7 +292,14 @@ def onFileNew(self):
 
     def onFileOpen(self):
         """Open a file."""
-        filename, _ = QFileDialog.getOpenFileName(self, "Open ipygraph from file")
+
+        filename, _ = QFileDialog.getOpenFileName(
+            self,
+            "Open ipygraph from file",
+            "",
+            "Ipygraph or Juyter notebook (*.ipyg *.ipynb)",
```
Comment (MathisFederico) [inline pyflow/graphics/window.py]:
```suggestion
            "Interactive python graph or notebook (*.ipyg *.ipynb)",
```

**Review by MathisFederico — CHANGES_REQUESTED** [review]
We still need to restrict what files can be opened in the first place, although a better tooltip is welcome !

**Comment by FabienRoger** [comment]
This does restrict the files that can be opened!

**Review by MathisFederico — APPROVED** [review]
Nice !

**Pull request merged**